README.md

```diff
@@ -8,3 +8,12 @@ Please add your schema to this repository under <code>gcn/notices/<i>mission</i>
 
 The GCN team is happy to iterate with the producers on their schema contents and format.
 Your pipeline will generate JSON files following these schema and send alerts to GCN as described in [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+
+## Version Updates
+
+New versions are simplified with the [`npm version`](https://docs.npmjs.com/cli/v8/commands/npm-version) command. The following steps describe the process for incrementing versions.
+
+1. Clone the parent repo (nasa-gcn/gcn-schema). This should not be run in your own forked version of this repo.
+
+2. Run `npm version [ major | minor | patch ]`
```

Comment:
The push commands are tied into the postversion script which runs automatically with the ncipollo/release-action@v1 action, but if we change this it will need to be updated